dynamiq/nodes/node.py

```diff
@@ -369,6 +410,8 @@ def transform_input(self, input_data: dict, depends_result: dict[Any, RunnableRe
             else:
                 inputs[key] = value
 
+        inputs = self.validate_input_schema(inputs)
```

Comment:
Here you have 2 ways:
1. Just use `self.validate_input_schema(inputs)` as still return dict. So you just going to validate and raise error if inputs are invalid. 
2. Transform inputs from `dict` to `input_schema` model by `inputs = self.validate_input_schema(inputs). But in this case all Nodes.execure with input_schema should handle this case. Also callbacks methods like `run_on_node_start` should handle `schema` too (not just `dict`)

Reply:
Should I stick to second approach?